src/app/pages/client-reviews/client-reviews.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="container reviews">
+    <div class="row">
+        <div class="col-md-8">
+            <div>
+                <h2>Client Details</h2>
+            </div>
+        </div>
+        <div class="example-container mt-3 col-md-4" style="max-width: 30rem;">
+
+            <div class="reviews" *ngIf="results?.length>0 else noreviews">
+                <hr>
+                <div class="text-center title"><b>{{count}} Reviews</b></div>
+                <hr>
+                <div class="mt-3" *ngFor="let review of results">
+                    <div class="reviewer">
+                        <img *ngIf="review.reviewer.image; else not" src={{review.reviewer.image}} class="profilepic" alt="avatar image">
+                        <ng-template #not>
```

Comment:
The template name here could be more descriptive... something like `#noImage` instead of `#not`. Makes the code easier to follow.